### PR TITLE
Complete Assignment 1.

### PR DESCRIPTION
**Pull request by CMUboy**
Michael, my Assignment 1 is completed.

### CODE REVIEW

**Comment by CMUboy** [comment]
Michael, my Assignment 2 is completed. assignment02-statement.html is the HTML output of the statement generated.

File: assignments/assignment01.rb [hunk 1]
```diff
@@ -37,13 +37,11 @@ def number_to_string(n, lang)
...
+# titleize "hEllo,   WORLD" => "Hello World"
 def titleize(s)
-  words = s.split
-  caps = []
-  words.each do |word|
-    caps << word.capitalize
-  end
-  caps.join " "
+  word_array = s.split(/\W+/)
+  (word_array.map { |w| w.capitalize }).join(" ")
```
Comment (planetcohen) [inline assignments/assignment01.rb:44]: No need for the parens, you can just write:
word_array.map { |w| w.capitalize }.join(" ")

File: assignments/assignment02.rb [hunk 2]
```diff
@@ -11,6 +14,9 @@
 
 def to_sentence(ary)
   # your implementation here
+  return ary.join("") if ary.length < 2
+  last_element = ary.pop
+  "#{ary.join(", ")} and #{last_element}"
```
Comment (planetcohen) [inline assignments/assignment02.rb:19]: perfect!

File: assignments/assignment02.rb [hunk 3]
```diff
@@ -26,10 +32,21 @@ def to_sentence(ary)
 # implement methods "mean", "median" on Array of numbers
 def mean(ary)
   # your implementation here
+  ary.reduce(:+) / ary.length
```
Comment (planetcohen) [inline assignments/assignment02.rb:35]: You need to convert one of them to floating point: either

ary.reduce(:+).to_f / ary.length

or

ary.reduce(:+) / ary.length.to_f

File: assignments/assignment02.rb [hunk 4]
```diff
@@ -26,10 +32,21 @@ def to_sentence(ary)
...
 end
 
 def median(ary)
   # your implementation here
+  # the array has to be sorted for the algorithm to work
+  sorted_ary = ary.sort()
```
Comment (planetcohen) [inline assignments/assignment02.rb:41]: in idiomatic ruby, you would not include the parens on the method call, instead you would just write:  sorted_ary = ary.sort

**Comment by planetcohen** [comment]
Overall, very nicely done!

**Pull request closed without merging**